ui/src/apps/clusterInfo/pages/List.js

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useState } from 'react'
-import { Tooltip, Popconfirm, Icon, Divider, Badge } from 'antd'
+import { DeleteOutlined } from '@ant-design/icons';
+import { Tooltip, Popconfirm, Divider, Badge } from 'antd';
```

Comment:
We need to keep the same coding style, for example, without semicolon at the end of the line.

Reply:
It must be `antd-codemod` add it. I'll fix it.